js/containers/portal-dashboard/feedback/activity-feedback-panel.tsx

```diff
@@ -1,22 +1,33 @@
 import React from "react";
+import { Map } from "immutable";
 import { connect } from "react-redux";
-import { updateActivityFeedback, updateActivityFeedbackSettings } from "../../actions/index";
-import { makeGetStudentFeedbacks, makeGetAutoScores, makeGetComputedMaxScore } from "../../selectors/activity-feedback-selectors";
+import { updateActivityFeedback, updateActivityFeedbackSettings } from "../../../actions/index";
+import { makeGetStudentFeedbacks, makeGetAutoScores, makeGetComputedMaxScore } from "../../../selectors/activity-feedback-selectors";
+import { ActivityLevelFeedbackStudentRows } from "../../../components/portal-dashboard/feedback/activity-level-feedback-student-rows";
+import { FeedbackLevel, ListViewMode } from "../../../util/misc";
 
 interface IProps {
+  activities: Map<any, any>;
   activity: Map<any, any>;
-  updateActivityFeedback: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
-  updateActivityFeedbackSettings: (activityId: string, activityIndex: number, feedbackFlags: any) => void;
+  activityIndex: number;
+  answers: any;
+  autoScores: any;
+  computedMaxScore: number;
+  currentStudentId: string | null;
+  currentQuestion: any;
+  feedbackLevel: FeedbackLevel;
   feedbacks: Map<any, any>;
   feedbacksNeedingReview: Map<any, any>;
-  numFeedbacksNeedingReview: number;
-  numFeedbacksGivenReview: number;
   feedbacksNotAnswered: number;
-  computedMaxScore: number;
-  autoScores: any;
-  settings: any;
+  isAnonymous: boolean;
+  listViewMode: ListViewMode;
+  numFeedbacksGivenReview: number;
+  numFeedbacksNeedingReview: number;
   rubric: any;
-  activityIndex: number;
+  settings: any;
+  students: Map<any, any>;
+  updateActivityFeedback: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
+  updateActivityFeedbackSettings: (activityId: string, activityIndex: number, feedbackFlags: any) => void;
 }
```

Comment:
There are a handful of `any` types in the props list that I wonder if we can find the proper typing for.

Reply:
There also appear to be some props that are not used that we could potentially remove.  With a quick search I saw that neither `activities` nor `currentStudentId` are used in this component.  Some of our props lists have become quite large.  If we can slim these down by removing unused props, that would be great.